### DISCUSSION TITLE
如何取消插入双链时，自动在标签处插入了链接

### DISCUSSION THREAD

**Discussion by GDSYY** [windingwind/zotero-better-notes#487] [Q&A] [answered]
这几天使用时，发现插入笔记时，页面变了，点击花朵插入到主笔记时，在文章对应的子笔记处自己添加了标签，结果就出现一堆不需要的标签。

![1](https://github.com/windingwind/zotero-better-notes/assets/134850091/4bbe3efd-e616-4d30-85aa-95ac13add274)
点击标签再返回后，笔记的按钮都消失了，重启软件才能恢复。
![image](https://github.com/windingwind/zotero-better-notes/assets/134850091/c483d08e-e622-4881-9599-ada97aefeca1)
![image](https://github.com/windingwind/zotero-better-notes/assets/134850091/335865bc-578d-4a40-943b-2133b6d6af62)
请问有什么办法解决这个问题？

**Comment by pkulhj** [comment]
顶顶！同问！怎么能不自动插入标签呀 TAT
模板编辑器看不懂 TAT

**Reply by Cynezhang** [reply]
一堆不必要的标签很苦恼

**Reply by windingwind** [reply, marked as answer]
这里的标签是用来再次点击花图表转跳已创建笔记的，最开始放在comment里，用户不满意所以放到标签里。如果不需要，不使用这个功能即可

**Reply by Cynezhang** [reply]
感谢回复！我刚刚试了一下这个功能，目前不是很需要，是直接把标签删除就可以了吗？

**Reply by windingwind** [reply]
是
